.travis.yml

```diff
@@ -15,10 +15,8 @@ install:
 
 env:
   matrix:
-   - DJANGO="Django<1.9"
-   - DJANGO="Django<1.10"
```

Comment:
Now that these versions were dropped, could you please remove the [`MiddlewareMixin` import exception handling](https://github.com/luizalabs/django-toolkit/blob/808c22d2bf2820f337c0b440ddd0b9ea12f284df/django_toolkit/middlewares.py#L9-L10)?

The code coverage reported a coverage drop, and it's because this line is not necessary anymore.